src/rust/madara/template/src/template.rs

```diff
@@ -66,12 +67,15 @@ impl Default for MadaraSiteData {
 			post_type: String::from("wp-manga"),
 			// p to select description from
 			description_selector: String::from("div.description-summary div p"),
+			// selector for chapter list
+			chapter_selector: String::from("li.wp-manga-chapter  "),
```

Comment:
I know it had the spaces previously but I'm pretty sure they're not necessary.
```suggestion
			chapter_selector: String::from("li.wp-manga-chapter"),
```